crates/aptos/src/common/utils.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{CliResult, Error};
+use serde::Serialize;
+
+/// Prompts for confirmation until a yes or no is given explicitly
+/// TODO: Capture interrupts
+pub fn prompt_yes(prompt: &str) -> bool {
+    let mut result: Result<bool, ()> = Err(());
+
+    // Read input until a yes or a no is given
+    while result.is_err() {
```

Comment:
my experience has been that a "no" is implicit and yes is explicit -- but maybe I've used TF too much lol

Reply:
It's just a loop until someone picks one or the other.  We could do implicit, I just stuck with explicit.